src/main/java/nl/bioinf/minorapplicationdesign/ontpillen/model/web_scraping/FarmacoWebScraper.java

```diff
@@ -32,9 +35,11 @@ public void setDrugDao(DrugDao drugDao) {
 
     @Override
     public void parseHtml() throws IOException {
-        System.out.println("basic url: " + this.basicUrl);
-        List drugs = Arrays.asList("Citalopram", "Lorazepam", "Temazepam");
-        this.parseInformation(drugs);
+        List<String> drugSubstances = new ArrayList<>();
+        for(Drug drugSubstance : drugDao.getDrugSubstances()){
+            drugSubstances.add(drugSubstance.getName());
+        }
```

Comment:
Hier hetzelfde als eerder, je kan volgens mij meteen een lijst terug krijgen. Ik vraag me ook af of er nog een aparte methode voor parseInformation moet zijn, kan dat neit gewoon in de parseHtml methode?